platform/ios/MGLMapboxEvents.m

```diff
@@ -144,6 +142,8 @@ @interface MGLMapboxEvents () <CLLocationManagerDelegate>
 @property (atomic) NSString *appName;
 @property (atomic) NSString *appVersion;
 @property (atomic) NSString *appBuildNumber;
+@property (atomic) NSNumber *version;
```

Comment:
It needs to be an Object and not just an NSUInteger otherwise Xcode complains.

![screen shot 2015-05-26 at 2 01 59 pm](https://cloud.githubusercontent.com/assets/989928/7823365/d0e63740-03af-11e5-9263-f8b96c84b97d.png)